Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -611,18 +611,30 @@ static std::optional<SRGBA<uint8_t>> parseNumericColor(std::span<const Character
             return *hexColor;
     }
 
-    // FIXME: rgb() and rgba() are now synonyms, so we should collapse these two clauses together. webkit.org/b/276761
-    if (mightBeRGBA(characters)) {
+    if (mightBeRGB(characters) || mightBeRGBA(characters)) {
         auto expectedUnitType = CSSUnitType::CSS_UNKNOWN;
+        auto current = mightBeRGBA(characters) ? characters.subspan(5) : characters.subspan(4); // handle 'a' for 'rgba()' case
 
-        auto current = characters.subspan(5);
+        // red and green will both terminate with ','
         auto red = parseColorIntOrPercentage(current, ',', expectedUnitType);
         if (!red)
             return std::nullopt;
         auto green = parseColorIntOrPercentage(current, ',', expectedUnitType);
         if (!green)
             return std::nullopt;
-        auto blue = parseColorIntOrPercentage(current, ',', expectedUnitType);
+
+        // check if blue terminates with ')' (no alpha-value provided)
```

Comment:
We generally format comments like a sentence https://webkit.org/code-style-guidelines/#comments-sentences